docs/examples/tiledata/color.js

```diff
@@ -0,0 +1,57 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import { TileData, Flex } from 'gestalt';
+
+export default function Example(): Node {
+  const [selectedTileId, setSelectedTileId] = useState('1');
+
+  const isSelected = (id: string) => selectedTileId === id;
+  const handleSelection = (ev) => {
```

Comment:
nit: this isn't actually returning an event, and there's no need for the local `ev` variable — just destructure what's needed